src/qrl/core/p2p/p2pChainManager.py

```diff
@@ -8,6 +8,12 @@
 
 
 class P2PChainManager(P2PBaseObserver):
+    """
+    P2PChainManager is a collection of function handlers that will be called
+    whenever P2P messages that have to do with blocks arrive. Such messages are
+    'fetch block', 'request block height', 'request headerhashes', 'block
+    received', and are defined in qrllegacy.proto:LegacyMessage.
```

Comment:
Comment for all the functions inside P2PChainManager that has been registered by Observable.register, will be more helpful.